pravega-cli/src/main/java/io/pravega/tools/pravegacli/commands/controller/ControllerDescribeStreamCommand.java

```diff
@@ -122,18 +131,42 @@ public static CommandDescriptor descriptor() {
     }
 
     private SegmentHelper instantiateSegmentHelper(CuratorFramework zkClient) {
+
+        // hostMonitorEnable should be true for pks clusters
         HostMonitorConfig hostMonitorConfig = HostMonitorConfigImpl.builder()
-                                                                   .hostMonitorEnabled(true)
+                                                                   .hostMonitorEnabled(false)
+                                                                   .hostContainerMap(getHostContainerMap(getCLIControllerConfig().getSegmentStoreURI(),
+                                                                           getServiceConfig().getContainerCount()))
                                                                    .hostMonitorMinRebalanceInterval(Config.CLUSTER_MIN_REBALANCE_INTERVAL)
                                                                    .containerCount(getServiceConfig().getContainerCount())
                                                                    .build();
         HostControllerStore hostStore = HostStoreFactory.createStore(hostMonitorConfig, StoreClientFactory.createZKStoreClient(zkClient));
         ClientConfig clientConfig = ClientConfig.builder()
                                                 .controllerURI(URI.create((getCLIControllerConfig().getControllerGrpcURI())))
-                                                .validateHostName(getCLIControllerConfig().isAuthEnabled())
-                                                .credentials(new DefaultCredentials(getCLIControllerConfig().getPassword(), getCLIControllerConfig().getUserName()))
+                                                .validateHostName(false)
```

Comment:
Why this is always false? Is should be as before, configurable depending on the deployment, right?

Reply:
Doesn't need to be always `false`. Have changed it to be configurable as it was before. The necessary change to incorporate this has been made when it comes to testing.